weave/integrations/openai/cassettes/openai_test/test_openai_async_quickstart.yaml

```diff
@@ -0,0 +1,1951 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate, br, zstd
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.31.0
+    method: GET
+    uri: https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json
```

Comment:
In this, and all other cassettes, please remove the `litellm` block. This is will reduce these files immensely